api-metastore/src/main/java/org/zalando/nakadi/controller/SchemaController.java

```diff
@@ -52,9 +63,25 @@ public ResponseEntity<?> create(@PathVariable("name") final String name,
             throw new ValidationException(errors);
         }
 
-        schemaService.addSchema(name, schema);
-
-        return status(HttpStatus.OK).build();
+        final var originalEventType = eventTypeService.fetchFromRepository(name);
+        schemaService.addSchema(originalEventType, schema)
+                .ifPresent(updatedEventType -> {
+                    nakadiAuditLogPublisher.publish(
+                            Optional.of(originalEventType),
+                            Optional.of(updatedEventType),
+                            NakadiAuditLogPublisher.ResourceType.EVENT_TYPE,
+                            NakadiAuditLogPublisher.ActionType.UPDATED,
+                            originalEventType.getName());
+
+                    nakadiKpiPublisher.publish(() -> new EventTypeLogEvent()
+                            .setEventType(name)
+                            .setStatus("updated")
+                            .setCategory(originalEventType.getCategory().name())
+                            .setAuthz(originalEventType.getAuthorization() == null ? "disabled" : "enabled")
+                            .setCompatibilityMode(originalEventType.getCompatibilityMode().name()));
+                });
+
+        return status(HttpStatus.OK).build(); // Maybe return different status code when there is no change
```

Comment:
Good point, we could use `201 Created` to indicate new schema version was really created.

Reply:
Looks like it's even documented that way: https://nakadi.io/manual.html#/event-types/name/schemas_post

Reply:
Created a new issue for addressing this.